src/i_oalmusic.c

```diff
@@ -429,6 +465,28 @@ static void *I_OAL_RegisterSong(void *data, int len)
                                             &player.freq, &player.frame_size))
         {
             active_module = all_modules[i];
+
+            int channels = AL_FORMAT_STEREO16;
```

Comment:
Did you mix up channels and format here?

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1997/commits/1922535a856d00b235cb8a9dc96747b285162e96